src/devices/ovrheadset/OVRHeadset.cpp

```diff
@@ -775,7 +775,7 @@ void yarp::dev::OVRHeadset::threadRelease()
         // Disable Performance Hud Mode before destroying the session,
         // or it will stay after the device is closed.
         int PerfHudMode = (int)ovrPerfHud_Off;
-        ovr_SetInt32(session, OVR_PERF_HUD_MODE, PerfHudMode);
+        ovr_SetInt(session, OVR_PERF_HUD_MODE, PerfHudMode);
```

Comment:
Those changes are not related to YARP 3, are those changed in the Oculus SDK? If they are, we should update also the `find_package` call...

Reply:
Some changes are related to yarp, the ones concerning rateThread and systemPeriodThread